hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/processors/CustomMessageCallProcessorTest.java

```diff
@@ -159,10 +160,11 @@ void callsToNonStandardSystemContractsAreNotSupported() {
         givenHaltableFrame(isHalted);
         givenCallWithCode(NON_EVM_PRECOMPILE_SYSTEM_ADDRESS);
         given(addressChecks.isSystemAccount(NON_EVM_PRECOMPILE_SYSTEM_ADDRESS)).willReturn(true);
+        when(frame.getValue()).thenReturn(Wei.of(2L));
```

Comment:
The _next_ test does a call-to-system-contract-with-value-transfer.  This test should be the call-to-system-contract-**without**-value-transfer one, right?  So it should mock `frame.getValue()` with `Wei.ZERO` and check for no-op success, I think?

Reply:
Good catch, ty